### PR TITLE
Update Italian langiuage files

### PR DESCRIPTION
**Pull request by bovirus**
@YerongAI 

Please merge. Thanks.

### CODE REVIEW

File: README-it_it.md [hunk 1]
```diff
@@ -21,31 +22,74 @@ Supporta la distribuzione personalizzata e l'attivazione dei seguenti prodotti:
...
 
-Oppure scarica Office Tool Plus utilizzando PowerShell:
+Oppure scarica Office Tool Plus usando PowerShell:
 
 ```powershell
 irm officetool.plus | iex
 ```
 
-## Informazioni
+## Grazie ai traduttori
```
Comment (YerongAI) [inline README-it_it.md:50]: There is no need to add this section. Only main README contain this.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 2]
```diff
@@ -212,121 +212,121 @@
...
+    <sys:String x:Key="Activate_Input_CID">Inserisci ID conferma</sys:String>
 
     <sys:String x:Key="Activate_KMS_Management">Gestione KMS</sys:String>
     <sys:String x:Key="Activate_KMS_Management.Description">Imposta o reimposta l'host o la porta del KMS.</sys:String>
     <sys:String x:Key="Activate_KMS_Apply">Impo_sta host</sys:String>
     <sys:String x:Key="Activate_KMS_Host">Host KMS</sys:String>
-    <sys:String x:Key="Activate_KMS_Host.Description">Nome host KMS. Per esempio, kms.example.com, kms.example.com:1688 o indirizzo IP.</sys:String>
-    <sys:String x:Key="Activate_KMS_Host_Cache.Enable">Abilita la memorizzazione nella cache dell'host</sys:String>
-    <sys:String x:Key="Activate_KMS_Host_Cache.Disable">Disabilita la memorizzazione nella cache dell'host</sys:String>
+    <sys:String x:Key="Activate_KMS_Host.Description">Nome host KMS. Per esempio, kms.esempio.com, kms.esempio.com:1688 o indirizzo IP.</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:306]: After thinking about it, I still think that domain names should not be translated. Keeping `kms.example.com` is better.

**Review by YerongAI — CHANGES_REQUESTED** [review]

File: README-it_it.md (hunk 1 above)
Comment (bovirus) [reply README-it_it.md]: @YerongAI 

Updated.
Please check if it's ok for you.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 2 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml]: @YerongAI 

Updated.
Please check if it's ok for you.

**Review by YerongAI — APPROVED** [review]

**Pull request merged**